Shared/ForChildren/ForChildrenItemView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ForChildrenItem.swift
+//  Movapp (iOS)
+//
+//  Created by Daryna Polevyk on 11.04.2022.
+
+import SwiftUI
+
+struct ForChildrenItemView: View {
+    let item: ForKidsItem
+    
+    var body: some View {
+        
+            VStack (spacing: 20) {
+                Image(item.image)
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .foregroundColor(.black)
+                VStack {
+                    HStack {
+                        Image("flagCzech")
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .frame(width: 30)
+                            .shadow(color: Color(red: 0, green: 0, blue: 0, opacity: 0.8), radius: 38, x: 0, y: 19)
+                        Text("česky")
+                            .font(.system(size: 20))
+                            .fontWeight(.bold)
+                        Spacer()
+                    }
+                    HStack {
+                        VStack(alignment: .leading) {
+                            Text(item.czTranslation)
+                                .foregroundColor(Color("colors/text"))
+                            Text("[\(item.czTranscription)]")
+                                .foregroundColor(Color("colors/secondary"))
+                        }
+                        Spacer()
+                        Image(systemName: "play.circle") //(systemName: soundService.isPlaying ? "stop.circle" : "play.circle")
+                            .resizable()
+                            .foregroundColor(Color("colors/action"))
+                            .frame(width: 30, height: 30)
+                            .onTapGesture {
+                                // soundService.speach(language: language, text: example.example)
```

Comment:
Tady dáš toto: (to language prosím předělat na constantu - můžeš myslím použít SetLanguage.csUk.language.from)

soundService.speach(language: 'cs', text: item.czTranslation)

Reply:
Reseno v separatni branchi https://github.com/cesko-digital/movapp-apple/pull/32